presto-verifier/src/main/java/com/facebook/presto/verifier/QueryRewriter.java

```diff
@@ -207,6 +244,13 @@ private static String dropTableSql(QualifiedName table)
         return formatSql(new DropTable(table, true), Optional.empty());
     }
 
+    private static String escapeLikeExpression(Connection connection, String value)
+            throws SQLException
+    {
+        String escapeString = connection.getMetaData().getSearchStringEscape();
+        return value.replaceAll("_", "\\" + escapeString + "_").replaceAll("%", "\\" + escapeString + "%");
```

Comment:
Why escape string is prefixed with a single backslash? None should be needed.

At least the escape string (or concatenation of it with "_") should be passed through `Matcher.quoteReplacement`.